src/main/java/org/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -1259,6 +1259,71 @@ public void combineCiteMarkers(List<BibDatabase> databases, OOBibStyle style)
         }
     }
 
+    /**
+     * Do the opposite of combineCiteMarkers.
+     * Combined markers are split, with a space inserted between.
+     */
+    public void unCombineCiteMarkers(List<BibDatabase> databases, OOBibStyle style)
+            throws IOException, WrappedTargetException, NoSuchElementException, IllegalArgumentException,
+            UndefinedCharacterFormatException, UnknownPropertyException, PropertyVetoException, CreationException,
+            BibEntryNotFoundException {
+        XNameAccess nameAccess = getReferenceMarks();
+        // TODO: doesn't work for citations in footnotes/tables
+        List<String> names = getSortedReferenceMarks(nameAccess);
+
+        final XTextRangeCompare compare = UnoRuntime.queryInterface(XTextRangeCompare.class, text);
+
+        int piv = 0;
```

Comment:
Please try to avoid abbreviations a variable names. Use full length names, maybe in camelCase.

Reply:
Do you mean piv should be pivot or pivotElement?
In https://github.com/antalk2/jabref/blob/ff1bc8933e3b05630633ef262e8b6f7928446ba9/src/main/java/org/jabref/gui/openoffice/OOBibBase.java#L1995 I renamed text to xtext, and expanded to this.xtext
to clarifiy its scope. The local variables compare and piv seemed OK.
If there is a part of the project that could serve as an example please point me there,
it might help to pick up the conventions.

Reply:
Yes, pivot is better. We usually also don't prefix variables with their type, so instead of `mxDocCursor` `documentCursor` (or `textCursor`?) would be preferred. Similarily, `bName ` below can simply be `name` or if you want to be verbose `markName`.